src/app/features/characters/services/marvel.service.ts

```diff
@@ -1,45 +1,30 @@
 import { Injectable } from '@angular/core';
 import { map, Observable } from 'rxjs';
 import { HttpClient } from '@angular/common/http';
-import { tap } from 'rxjs';
-
-
-interface MarvelApiResponse {
-  data: {
-    results: any[];
-  };
-}
-
-interface MarvelApiCharResponse {
-  data: {
-    results: any;
-  };
-}
-
+import { MarvelApiCharsResponse, MarvelChar } from '@characters/models/characters.model';
 
 @Injectable({
   providedIn: 'root'
 })
-export class MarvelService {
-  apiBase = 'https://gateway.marvel.com:443/v1/public/';
-  apiKey = 'apikey=fe7700287fdcc6bdb92ae7f6e5f64d9c';
+export class CharactersMarvelService {
+  charsLimit: number = 9;
```

Comment:
if you are going to add one more service and the limin and the offset are the same, it might be a good idea to move the to some global constants

Reply:
I will add at least one more service, but these variables will have different value